pages/dkp/kommander/2.2/networking/use-istio/index.md

```diff
@@ -0,0 +1,114 @@
+
+This topic describes how to expose an application running on the DKP cluster using the LoadBalancer (layer-4) service type.
+
+<p class="message--note"><strong>NOTE: </strong>Usage and installation of Istio on DKP is currently a self-service feature.</p>
+
+## Before you begin
+Before you begin, verify the following:
+
+- You must have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You must have a properly deployed and running cluster.
+
+## Deploy Istio using DKP
+
+Review the [list of available applications](../../workspaces/applications/platform-applications#workspace-platform-applications) to obtain the current 'APP NAME' for the Istio application, as you will need this name later in this procedure.
+
+1.  Enable the deployment of Istio to [your existing attached cluster](../../clusters/attach-cluster/) by creating an `AppDeployment` resource.
+
+1.  Within the `AppDeployment`, define the `appRef` to specify which `App` will be enabled:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps.kommander.d2iq.io/v1alpha2
+    kind: AppDeployment
+    metadata:
+      name: istio
+      namespace: ${WORKSPACE_NAMESPACE}
+    spec:
+      appRef:
+        name: istio-1.11.6
+        kind: ClusterApp
+    EOF
+    ```
+
+1.  Create the resource in the workspace you just created, which instructs Kommander to deploy the `AppDeployment` to the `KommanderCluster`s in the same workspace.
+
+<p class="message--note"><strong>NOTE: </strong>The <code>appRef.name</code> must match the app <code>name</code> from the list of available applications.</p>
+
+## Download the Istio command line utility
+
+1. Store a local environment variable containing the current Istio version running in your cluster:
+
+    ```bash
+    export KONVOY_ISTIO_VERSION="$(kubectl get clusteraddons istio -o go-template='{{ .spec.chartReference.version }}')"
+    ```
+
+2. Pull a copy of the corresponding Istio command line to your system:
+
+    ```bash
+    curl -L https://istio.io/downloadIstio | ISTIO_VERSION=${KONVOY_ISTIO_VERSION} sh -
+    ```
+
+3. Change to the Istio directory and set your PATH environment variable by running the following commands:
+
+    ```bash
+    cd istio*
+    export PATH=$PWD/bin:$PATH
+    ```
+
+4. You should now be able to run the following `istioctl` command and view the subsequent output:
+
+    ```bash
+    istioctl version
+    ```
+    
+    ```sh
+    client version: <your istio version here>
+    control plane version: <your istio version here>
+    data plane version: <your istio version here> (1 proxies)
+    ```
+
+## Deploy a sample application on Istio
+
+The Istio BookInfo sample application is composed of four separate microservices that demonstrate various Istio features.
+
+1. Deploy the sample `bookinfo` application on the Kubernetes cluster by running the following commands:
```

Comment:
these commands probably need to be run on the attached/managed clusters where istio got deployed right? might want to specify or add a line about using the correct kubeconfig context